doc/man7/EVP_KEM-MLKEM.pod

```diff
@@ -0,0 +1,63 @@
+=pod
+
+=head1 NAME
+
+EVP_KEM-ML-KEM-512,EVP_KEM-ML-KEM-768,EVP_KEM-ML-KEM-1024,EVP_KEM-MLKEM
+- EVP_KEM ML-KEM keytype and algorithm support
+
+=head1 DESCRIPTION
+
+The B<ML-KEM> keytypes and parameters are described in L<EVP_PKEY-MLKEM(7)>.
+See L<EVP_PKEY_encapsulate(3)> and L<EVP_PKEY_decapsulate(3)> for more info
+regarding the basic KEM operations.
+
+=head2 ML-KEM KEM parameters
+
+=over 4
+
+=item "entropy" (B<OSSL_KEM_PARAM_MLKEM_ENC_ENTROPY>)<UTF8 string>
+
+The OpenSSL ML-KEM encapsulation mechanism can only be modified by
+setting randomness during encapsulation facilitating testing as per
+FIPS 203, section 6.2, algorithm 17.
```

Comment:
Encapsulation normally consumes 32 bytes of RNG *entropy*, which is hashed to produce other random values within the encapsulation process.  Specifying this explicitly, instead of generating it randomly, make encasulation fully deterministic.